examples/kitchensink/index.js

```diff
@@ -315,14 +315,23 @@ function handleAudio(message, replyToken) {
 
   return downloadContent(message.id, downloadPath)
     .then((downloadPath) => {
-      return client.replyMessage(
-        replyToken,
-        {
-          type: 'audio',
-          originalContentUrl: baseURL + '/downloaded/' + path.basename(downloadPath),
-          duration: 1000,
-        }
-      );
+      // get-audio-duration is needed here to get audio duration
+      // npm install get-audio-duration --save
```

Comment:
Instead of this comment, I think you can add `get-audio-duration` to `package.json` and `package-lock.json` using the command you mentioned in the comment.